compiler/ras/Debug.hpp

```diff
@@ -426,9 +426,6 @@ class TR_Debug
    virtual void         printHeader();
    virtual void         printMethodHotness();
    virtual void         printInstrDumpHeader(const char * title);
-#if defined(TR_TARGET_ARM)
-   virtual void         printAsmDumpHeader();
-#endif
```

Comment:
Please update the copyright header in this file.

Reply:
Updated the copyright year of Debug.hpp.